lib/core/src/sdk.rs

```diff
@@ -1740,12 +1740,27 @@ impl LiquidSdk {
     /// List all failed chain swaps that need to be refunded.
     /// They can be refunded by calling [LiquidSdk::prepare_refund] then [LiquidSdk::refund].
     pub async fn list_refundables(&self) -> SdkResult<Vec<RefundableSwap>> {
-        Ok(self
-            .persister
-            .list_refundable_chain_swaps()?
-            .into_iter()
-            .map(Into::into)
-            .collect())
+        let chain_swaps = self.persister.list_refundable_chain_swaps()?;
+
+        let mut refundables = vec![];
+        for chain_swap in chain_swaps {
+            let script_pubkey = chain_swap.get_lockup_swap_script_pubkey(self.config.network)?;
+            let script_balance = self
+                .bitcoin_chain_service
+                .lock()
+                .await
+                .script_get_balance(script_pubkey.as_script())?;
```

Comment:
Electrum has a batched version of this API: https://docs.rs/electrum-client/latest/electrum_client/trait.ElectrumApi.html#tymethod.batch_script_get_balance
Perhaps we can use get all balances in one call.

Reply:
Done.